qemu/tests/qmp_command.py

```diff
@@ -218,6 +219,18 @@ def qmp_cpu_check(output):
                                               "or higher than no_cpus).\n%s"
                                               % (cpu, out))
 
+    def qmp_pmemsave_check(qmp_cmd, post_check_o, post_o):
+        """ pmemsave test check """
+        size_input = int(qmp_cmd.split(',')[1].split('=')[1])
+        size_output = int(post_check_o)
+        if os.path.exists(post_o):
+            if size_output == size_input:
+                pass
+            else:
+                test.fail("pmemsave file size should %d,  but size is %d" % (size_input, size_output))
+        else:
+            test.fail("pmemsave file didn't exists!")
```

Comment:
`if size_output != size_input:`
     `test.fail("pmemsavefile size should %d,  but size is %d" % (size_input, size_output))`
It does the same thing and this could save you 2 lines.
There are two grammar  mistakes: 1) pmemsavefile size should be 2) pmemsave file didn't exist !